VersionManager.py

```diff
@@ -51,10 +51,14 @@
                   "tex/gregoriotex.tex",
                   "tex/gregoriotex-chars.tex",
                   "tex/gregoriotex-signs.tex",
+                  "tex/gregoriotex-signs.lua",
                   "tex/gregoriotex-spaces.tex",
                   "tex/gregoriotex-symbols.tex",
+                  "tex/gregoriotex-symbols.lua",
                   "tex/gregoriotex-syllable.tex",
                   "tex/gregoriotex-main.tex",
+                  "tex/gregoriotex-nabc.tex",
+                  "tex/gregoriotex-nabc.lua",
```

Comment:
Of all the things for me to use as a search term to find where `gregoriotex-symbols.lua` was missing, I chose `gregoriotex-nabc.lua`, which of course missed this file.  Thanks for finding it.